Add the use of Emma to Reinstalling Sailfish OS document (#287)
sailfishos/docs.sailfishos.org: Support/Help_Articles/Reinstalling_Sailfish_OS/README.md

# Reverting Xperia device to Android OS and reinstalling Sailfish OS

It has turned out that the following procedure may sometimes help if something has gone wrong while preparing the device for Sailfish OS and/or while installing Sailfish OS.

These instructions are applicable for all [Xperia models supported by Sailfish OS](/Support/Supported_Devices/).

Depending on the problems observed on your Xperia, it may be enough to just **reinstall ("reflash") Sailfish OS** to it. It is good to try this easier way first. In this case, skip to chapter "[Installing Sailfish X again to Xperia](https://docs.sailfishos.org/Support/Help_Articles/Reinstalling_Sailfish_OS/#installing-sailfish-x-again-to-xperia)" in this document.

However, if your Xperia cannot use the services of cellular network properly (or at all), then it may be necessary to revert the device back to Android OS first, and finally, reinstall Sailfish X.

**NOTE:** _Reverting the device back to Android means that **all data** in the device memory **will be deleted**. Only the SD and SIM cards remain untouched._

# This is our plan

1.  Revert the device back to Android OS first

> 2.  Use your device and your SIM card with Android OS, checking the main functionality (see chapter [4 Checking that the phone works with Android](#checking-that-the-phone-works-with-android))

Reviewer: I'd drop the chapter number from the link. If the numbering changes in the future, there would be one less thing to fix.

Author: Chapter number removed.

3.  Re-flash Sailfish OS to the device.

# Reverting Xperia back to Android OS

## Getting Emma and connecting Xperia to it

Reverting Xperia device back to Android requires the **[Emma](https://developer.sony.com/develop/open-devices/get-started/flash-tool/download-flash-tool/)** flashing tool from Sony. Emma runs on **Windows PC** only. It does not work on Linux or Mac computers.

1. Download and install the Flash tool “Emma”  from **[Sony’s developer website](https://developer.sony.com/develop/open-devices/get-started/flash-tool/download-flash-tool/)** to your Windows computer.
2. Turn off your Xperia device. Leave it off for at least fifteen (15) seconds to be sure it is off.
3. Hold down the **Volume Down** button on the device while connecting your device to your computer with a USB **data** cable. The LED next to the speaker on the device should soon turn from red to **green** (not blue).

> 4. Run the Emma tool. A view similar to that in Picture 3a should appear. It shows the phone model on the left. 

Reviewer: It would be easier to follow if "Picture 3a" was a link to the picture. Just add `name="picture_3a"` to the anchor tag there, and change this one to a link.

Author: Oh boy. I did not realize this is possible. And I have added trillions of pictures to other docs with verbal links to them. Sigh.

Reviewer: I know, I have been reviewing those. I'd say about 99% of cases the picture is right next to the text, so adding a link is not really necessary. Here the picture is a bit further away, so adding a link would be a bigger improvement.

> 4. Run the Emma tool. A view similar to that in Picture 3a should appear. It shows the phone model on the left. 

Reviewer: ```suggestion
4. Run the Emma tool. A view similar to that in [Picture 3a](#picture_3a) should appear. It shows the phone model on the left. 
```

## Selecting the version of Android baseband

See Picture 3a below. It shows the Emma home view on your PC once you have connected your Xperia phone (here: Xperia 10 II). The current Android baseband version (a.k.a. build number) is shown in the "Software" box under the phone icon in the Emma view.

Check the latest compatible Android baseband version (or "service" as they are called in Emma) according to the **[baseband version table](#the-baseband-version-table)** below. The corresponding Android OS versions shown in the phone settings are in parentheses. Pay attention to the first 2 numbers (preceeding the letter A).

<div class="flex-images" markdown="1">

> * <a href="Emma_X10II_home.png"><img src="Emma_X10II_home.png" alt="Emma home view"></a>

Reviewer: ```suggestion
* <a href="Emma_X10II_home.png" name="picture_3a"><img src="Emma_X10II_home.png" alt="Emma home view"></a>
```
Make this image "linkable"

  <span class="md_figcaption">

>     Picture 3a: Initial view of Emma. Xperia 10 II connected.

Reviewer: Now somehow I find it a bit weird to start numbering from "3a". I would have started from 1 ;)

Author: The picture numbering was the same as the chapter numbering. But I'll change it.

  </span>
</div>
  

> As of March 2023, the latest services were as below but note that the exact version ID's can vary by model (regular/Plus/Ultra) and by sales area. The first part of the ID (in bold letters) is significant and should be followed (anything after the letter A may vary and does not matter in this context). This table contains the baseband versions that Jolla has used when testing the Sailfish port. If you selected a more recent version then you take a risk that not everything works as expected..

Reviewer: Now there's an extra full stop at the end

Author: Removed.

> As of March 2023, the latest services were as below but note that the exact version ID's can vary by model (regular/Plus/Ultra) and by sales area. The first part of the ID (in bold letters) is significant and should be followed (anything after the letter A may vary and does not matter in this context). This table contains the baseband versions that Jolla has used when testing the Sailfish port. If you selected a more recent version then you take a risk that not everything would work.

Reviewer: ```suggestion
As of March 2023, the latest services were as below but note that the exact version ID's can vary by model (regular/Plus/Ultra) and by sales area. The first part of the ID (in bold letters) is significant and should be followed (anything after the letter A may vary and does not matter in this context). This table contains the baseband versions that Jolla has used when testing the Sailfish port. If you selected a more recent version then you take a risk that not everything works as expected.
```

Author: Language improved.

### The baseband version table

| Device        | Android baseband version                                                              |
| ------------- | ---------------------------------------------------------------------------- |
| Xperia X      | **34.4**.A.2.118 (Android 8)                                                 |
| Xperia XA2    | **50.2**.A.0.400 (Android 9) [1] <br /> **50.1**.A.13.123 (Android 8) [2]    |
| Xperia 10     | **53.0**.A.8.69 (Android 9) [3]                                              |
| Xperia 10 II  | **59.1.**A.2.169 (Android 11) <br /> **59.0**.A.6.24 (Android 10) [4]        |
| Xperia 10 III | **62.0.**A.3.109 (Android 11) <br /> **62.1.**A (Android 12) [5]             |

> _[1] This Android 9 version appears to sometimes get an invalid WiFi MAC which prevents the phone from using WiFi networks (in the Android mode). In such a case, first flash an Android 8 version, boot up, and then flash Android 9.  Note that even if there was no WiFi on Android 9, after installing Sailfish, WiFi works again._

Reviewer: How about proper MD footnotes? Or at the very least, add linebreaks.

Author: Footnotes applied.
